diagnostic.md

```diff
@@ -36,13 +39,16 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET 'http://www.example.com/donuts/34'
 ```
 
 Write an AJAX request to retrieve a **single donut** from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut = $.ajax({
+  url: 'http://example.com/donuts/34',
+  method: 'GET',
+});
 ```
```

Comment:
concatenating the id would make more sense here, but this would work.

```js
let getDonut = function (donutID) {
  return $.ajax({
    url: 'http://www.example.com/donuts/' + donutID,
    method: 'GET',
  });
};
```